src/themes/teams/components/Text/textStyles.ts

```diff
@@ -27,7 +27,15 @@ export default {
   }: TextStylesParams): ICSSInJSStyle => {
     return {
       ...(truncated && truncateStyle),
-      ...(atMention && { color: v.atMentionTextColor }),
+      ...(atMention &&
+        atMention === 'other' && {
+          color: v.atMentionOtherTextColor,
+        }),
+      ...(atMention &&
+        atMention === 'me' && {
```

Comment:
No need to check for `atMention &&` first.  If there is no `atMention` then the next logical check `atMention === 'me'` will fail.  It is sufficient to only have this one check.